packages/app/src/components/inspector/SymbolLayoutRow.tsx

```diff
@@ -0,0 +1,174 @@
+import { memo, useCallback } from 'react';
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface Props {
+  groupLayout?: Sketch.FreeformGroupLayout | Sketch.InferredGroupLayout;
+  setLayoutAxis: (value: Sketch.InferredLayoutAxis | undefined) => void;
+  setLayoutAnchor: (value: Sketch.InferredLayoutAnchor) => void;
+  setMinWidth: (value: number, mode: 'replace' | 'adjust') => void;
+}
+
+export default memo(function SymbolInspector({
+  groupLayout,
+  setMinWidth,
+  setLayoutAxis,
+  setLayoutAnchor,
+}: Props) {
+  const inferredLayout =
+    groupLayout && Layers.isInferredLayout(groupLayout)
+      ? groupLayout
+      : undefined;
+
+  const layoutAxis = inferredLayout?.axis;
+  const layoutAnchor = inferredLayout?.layoutAnchor;
+  const minWidth = inferredLayout?.minSize;
+  const isAnchorDisabled = layoutAxis === undefined;
+
+  const renderLabel = useCallback(({ id }) => {
+    switch (id) {
+      case '0-0':
+        return <Label.Label>Left to Right</Label.Label>;
+      case '0-1':
+        return <Label.Label>Center</Label.Label>;
+      case '0-2':
+        return <Label.Label>Right to Left</Label.Label>;
+      case '1-0':
+        return <Label.Label>Top to Bottom</Label.Label>;
+      case '1-1':
+        return <Label.Label>Middle</Label.Label>;
+      case '1-2':
+        return <Label.Label>Bottom to Top</Label.Label>;
+      default:
+        return <Label.Label></Label.Label>;
+    }
+  }, []);
+
+  const handleNudgeMinWidth = useCallback(
+    (value: number) => setMinWidth(value, 'adjust'),
+    [setMinWidth],
+  );
+
+  const handleSetMinWidth = useCallback(
+    (value) => setMinWidth(value, 'replace'),
+    [setMinWidth],
+  );
+
+  return (
+    <InspectorPrimitives.Section>
+      <InspectorPrimitives.Title>Layout</InspectorPrimitives.Title>
+      <Spacer.Vertical size={4} />
+      <InspectorPrimitives.Row>
+        <RadioGroup.Root
+          value={layoutAxis?.toString() ?? 'none'}
+          onValueChange={useCallback(
+            (event) =>
+              setLayoutAxis(
+                event.target.value === 'none'
+                  ? undefined
+                  : parseInt(event.target.value),
+              ),
+            [setLayoutAxis],
+          )}
+        >
+          <RadioGroup.Item value={'none'}>None</RadioGroup.Item>
+          <RadioGroup.Item
+            value={Sketch.InferredLayoutAxis.Horizontal.toString()}
+          >
+            Horizontal
+          </RadioGroup.Item>
+          <RadioGroup.Item
+            value={Sketch.InferredLayoutAxis.Vertical.toString()}
+          >
+            Vertical
+          </RadioGroup.Item>
+        </RadioGroup.Root>
+      </InspectorPrimitives.Row>
+      <Spacer.Vertical size={6} />
+
+      <InspectorPrimitives.Row>
+        <LabeledElementView renderLabel={renderLabel}>
+          <RadioGroup.Root
+            id={`${layoutAxis}-${layoutAnchor}`}
+            value={layoutAnchor?.toString() ?? 'none'}
```

Comment:
I think it's good like this, but I still like the idea of using `Horizontal` as a default, since visually that's what we show even when "none" is selected for the axis. Can we do the "none" based on `isAnchorDisabled?` E.g. 

```ts
value={isAnchorDisabled ? undefined : layoutAnchor.toString()}
```

The `disabled` prop should already prevent us from calling the callback with an invalid value. I think we can also use `undefined` instead of `"none"` in this case, since we don't need to show a `"none"` RadioGroup.Item

Reply:
`isAnchorDisabled checks if layoutAxis === undefined` if I do this `const layoutAxis =
    inferredLayout?.axis ?? Sketch.InferredLayoutAxis.Horizontal;` it will never be false 🤔

Reply:
using inferredLayout  instead of the layoutAxis fixes this issue  👍

Reply:
oh if i use undefined instead of a string the  radio does this weird thing were it just puts a random value 🤔 and selects something even when it shouldn't
with `undefined` (`isAnchorDisabled ? undefined : layoutAnchor?.toString()`)
![image](https://user-images.githubusercontent.com/16924758/119578399-132c4a80-bd82-11eb-8775-549284b3f45a.png)
putting a string (`isAnchorDisabled ? '' : layoutAnchor?.toString()`)
![image](https://user-images.githubusercontent.com/16924758/119578407-18899500-bd82-11eb-87e5-7382d4672818.png)